apps/docs/content/guides/auth/auth-smtp.mdx

```diff
@@ -1,38 +1,136 @@
 ---
 id: 'auth-smtp'
-title: 'Configure a Custom SMTP'
+title: 'Send emails with custom SMTP'
 description: 'Moving towards production: Configuring a custom SMTP provider'
 ---
 
-## Auth SMTP
+If you're using Supabase Auth with the following configuration:
 
-At present, you can trial the Supabase platform by sending up to **3** emails per hour via the built-in service. The default email service as a whole is offered on a best effort basis: we will do our best to maintain it and will review usage of the service on a regular basis to see if the email service should be continued.
+- Email and password accounts
+- Passwordless accounts using one-time passwords or links sent over email (OTP, magic link, invites)
+- Email-based account invitations from the Supabase dashboard or from the Auth admin APIs
+- Social login with email confirmation
 
-As you progress toward production, you may find yourself wanting for a custom SMTP service in order to increase your limits. A custom SMTP server will allow you to set your own cap on the number of emails sent per hour.
+You will need to set up a custom SMTP server to handle the delivery of messages to your users.
 
-Beyond rate limits, an SMTP server might also help with:
+To get you started and let you explore and set up email message templates for your application, Supabase provides a simple SMTP server for all projects. This server imposes a few important restrictions and is not meant for production use.
 
-- Deliverability and Reputation Management
-- Scalability
-- Analytics and Tracking
-- Compliance and Anti Spam measures
+**Send messages only to pre-authorized addresses.**
 
-## How to set up SMTP
+Unless you configure a custom SMTP server for your project, Supabase Auth will refuse to deliver messages to addresses that are not part of the project's team. You can manage this in the [Team tab](https://supabase.com/dashboard/org/_/team) of the organization's settings.
 
-Head over to [Auth Settings Page](https://supabase.com/dashboard/project/_/settings/auth) and hit "Enable Custom SMTP" under the SMTP Provider section.
+For example, if your project's organization has these member accounts `person-a@example.com`, `person-b@example.com` and `person-c@example.com` then Supabase Auth will only send messages to these addresses. All other addresses will fail with the error message _Email address not authorized._
 
-Fill in fields below with the relevant details obtained from your custom SMTP provider:
+**Significant rate-limits that can change over time.**
 
-![SMTP settings](/docs/img/guides/auth-smtp/smtp.png)
+To maintain the health and reputation of the default SMTP sending service, the number of messages your project can send is limited and can change without notice. Currently this value is set to 2 messages per hour.
```

Comment:
Consider using shared constant so it doesn't of sync

Reference: https://github.com/supabase/supabase/blob/0aa4793de8bb9838419b014d35bea73a3dc527bb/apps/docs/components/MDX/auth_rate_limits.mdx?plain=1#L3
```suggestion
To maintain the health and reputation of the default SMTP sending service, the number of messages your project can send is limited and can change without notice. Currently this value is set to <SharedData data="config">auth.rate_limits.email.inbuilt_smtp_per_hour</SharedData> messages per hour.
```